### PR TITLE
refactor(nexus): clean up

### PR DESCRIPTION
**Pull request by kptkin**
copilot:summary

### CODE REVIEW

File: nexus/pkg/server/handler.go [hunk 1]
```diff
@@ -155,103 +94,132 @@ func (h *Handler) handleRequest(rec *service.Record, req *service.Request) {
...
+	}
+
+	response.ResponseType = &service.Response_GetSummaryResponse{GetSummaryResponse: &service.GetSummaryResponse{Item: items}}
+}
+
+func (h *Handler) handleDefer(rec *service.Record, req *service.DeferRequest) {
+	// switch req.State {
+	// default:
+	// 	h.sendRecord(rec)
+	// }
```
Comment (kptkin) [inline nexus/pkg/server/handler.go:177]:
```suggestion
```

File: nexus/cmd/nexus/main.go [hunk 2]
```diff
@@ -17,7 +18,8 @@ func main() {
 
 	flag.Parse()
 
-	server.SetupLogger(true)
```
Comment (kptkin) [inline nexus/cmd/nexus/main.go]: why not use another flag?

Comment (kptkin) [reply nexus/cmd/nexus/main.go]: don't mind either way...

Comment (raubitsj) [reply nexus/cmd/nexus/main.go]: flag is hard to use since it is launched from python.. i think this is fine for now.

**Pull request merged**